### ISSUE TITLE
模型FTIA跑出来的效果不能够实现文本分类

### ISSUE THREAD

**Issue by Vivi-cookie** [bannima/FTIA#1]
您好，我试着运行了您发布的代码，发现FTIA模型运行的效果并没有成功完成文本分类，而是堆积在了一起，想请教下您觉得这可能是会什么原因造成的呢？

**Comment by bannima** [comment]
能否给一下你复现实验的细节？还有堆积在一起是tsne降维后可视化效果不明显的意思么？

**Comment by Vivi-cookie** [comment]
![image](https://user-images.githubusercontent.com/65707124/110723756-81677380-824f-11eb-8944-fd7b4e21bb3d.png)
这是我没有修改FTIA模型时，于TREC数据集上跑时第1个epoch和第50个epoch的可视化效果图基本相同，如上图所示点的分布很分散。
当我将batch_size改为10，修改FTIA中这一部分：requires_grad=False后
if word_vector is not None:
            self.word_embedding.weight = nn.Parameter(word_vector, requires_grad=False)
于TREC数据集上第一个epoch可视化效果如下，但第70个epoch效果也几乎相同，没有分类
![image](https://user-images.githubusercontent.com/65707124/110724434-b7f1be00-8250-11eb-8427-b47b9f407b8d.png)
其余代码并没有太多改动，然后还有一个小疑问是我在SST1和CR数据集上运行LSTMAtt模型上都会出现IndexError:index 21524 is out of bounds for axis 0 with axis 0 with size 21308；但在TREC数据集上并不会出现。会是因为数据集大小的原因吗，该怎么解决比较好呢？

**Comment by bannima** [comment]
第一幅图，图中数据点很少，严重怀疑是哪里有问题，第二幅图也很奇怪，图中的点也不会重合的这么好，我的实验结果都是readme中的图，而且跑了很多次都没有问题，不仅仅是tsne降维后的结果清晰，而且也有实验准确率的支持，请问你最终跑出来的准确率是正常的么？github消息太慢，有需要加微信zhouenguo详细交流吧
